### DISCUSSION TITLE
Project maintenance

### DISCUSSION THREAD

**Discussion by ruanspies** [go-git/go-git#687] [General]
Good morning,

We have really appreciated the work done in this project - thanks to everyone who has been part of building it out. 

It does however seem that the project has lost a bit of momentum given the number of open issues and PRs.

It would be great to have some information around the maintenance of the project. The main page mentions that it is 'actively maintained' but things seem to happen quite slowly. What are the avenues for the community to get involved and pick up momentum with this project again? Myself and my team would be keen to get involved.

@mcuadros @jfontan

**Comment by pjbgf** [comment]
+1 on keenness to help support this project.

**Comment by mcuadros** [comment]
Here is a small explanation of the current situation: https://github.com/go-git/go-git/issues/678#issuecomment-1449040634

Obviously, I am open to letting some active contributors lead the project.

**Comment by aymanbagabas** [comment]
Since I've been working so closely with the git ecosystem, I'm willing to help maintain this awesome project :)
It would be great to have a communication channel. Many go projects have a dedicated discord channel.

**Comment by pjbgf** [comment]
I created a Slack workspace for go-git, so contributors can bounce some ideas when needed. Here's the invitation link: https://join.slack.com/t/go-git/shared_invite/zt-1tk7vwrsw-ywCUKRCo_uS_QbKMQ2ukpA

**Reply by aymanbagabas** [reply]
Moved to Discord https://discord.gg/6CG7M6wF

**Reply by damsien** [reply]
> Moved to Discord https://discord.gg/6CG7M6wF

The link is not correct anymore, here is the right one: https://discord.gg/8hrxYEVPE5 (from slack)
